packages/terra-modal/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "react"],
+  "plugins": ["transform-object-assign", "transform-object-rest-spread", "transform-runtime"],
```

Comment:
Transform-runtime is necessary to support Set.  I tried to scope this plugin just to terra-modal but I couldn't scope it just to transform Set.  Unfortunately it does a few other things as well.  Are we OK with this solution or should we investigate an alternative to Set?